src/cleanlab_tlm/utils/chat.py

```diff
@@ -442,8 +442,12 @@ def form_prompt_string(
         else _form_prompt_chat_completions_api(cast(list["ChatCompletionMessageParam"], messages), tools)
     )
 
+def form_response_string_chat_completions_api(response: "ChatCompletion") -> str:
+    """ Form a single string representing the response, out of the raw response object returned by OpenAI's Chat Completions API."""
+    response_msg = response.choices[0].message
```

Comment:
i'm unsure if this is always the only way to get the response-message, or there might be other ways sometimes (eg. structured outputs, tool calls, etc)

Reply:
The underlying `ChatCompletionMessage` may look different when using structured outputs, tool calls, and similar features.

We can still have `len(response.choices) > 1` when `openai_kwargs["n"] > 1`, so it’s important to clarify in the documentation how that kwarg is handled.